include/gatekeeper_acl.h

```diff
@@ -64,7 +64,8 @@ process_pkts_acl(struct gatekeeper_if *iface, unsigned int lcore,
 	switch (proto) {
 	case ETHER_TYPE_IPv4:
 		return process_acl(iface, lcore, acl, &iface->ipv4_acls,
-			ipv4_if_configured(iface), "IPv4");
+			!iface->hw_filter_ntuple && ipv4_if_configured(iface),
```

Comment:
I recommend adding a comment here explaining the new test since it now diverges from the IPv6 test and it's not obvious from the surroundings of the code the why. The comment would be just an adaptation of the comment of the patch.